### DISCUSSION TITLE
No way to remove build-in imports like CodeView / LottieAnimation

### DISCUSSION THREAD

**Discussion by Maks-gaming** [Sketchware-Pro/Sketchware-Pro#575] [Q&A] [answered]
There is no way for users to remove built-in component imports in case they need to supply their own with the same class name. This issue is likely to be of great importance to users.

Maybe it is high-priority issue?

**Comment by j-px** [comment]
Don't Java Command Blocks work?

**Comment by Maks-gaming** [comment]
> Don't Java Command Blocks work?

No, if i'll add for example custom codeview lib with the same classname, then a class conflict error will appear

**Comment by Maks-gaming** [comment]
Conflict with built-in codeview, but its not used
![Screenshot_20220907-085148_Sketchware Pro.jpg](https://user-images.githubusercontent.com/77161824/188798968-ab0e2e6b-bad5-41f7-88e5-86b4320d0dd8.jpg)

**Comment by j-px** [comment, marked as answer]
You can remove automatically added imports like this:

![removeImport](https://user-images.githubusercontent.com/40118124/188819982-bfc6da2d-a1bf-43a8-afd9-8906679ff8e1.png)

**Comment by Maks-gaming** [comment]
> You can remove automatically added imports like this:
> 
> ![removeImport](https://user-images.githubusercontent.com/40118124/188819982-bfc6da2d-a1bf-43a8-afd9-8906679ff8e1.png)
> 
> 

Where can I find this block?

**Comment by Maks-gaming** [comment]
nvm, found it, but still dont understand how to use it

**Comment by j-px** [comment]
There are tutorials on Command Blocks in our Discord server in channel #tips-n-tricks.
